internal/command/machine/update.go

```diff
@@ -106,6 +106,8 @@ func runUpdate(ctx context.Context) (err error) {
 	fmt.Fprintln(out, colorize.Yellow(fmt.Sprintf("Machine %s has been updated\n", machine.ID)))
 	fmt.Fprintf(out, "Instance ID has been updated:\n")
 	fmt.Fprintf(out, "%s -> %s\n\n", prevInstanceID, machine.InstanceID)
+	fmt.Fprintln(out, "The following config has been updated")
```

Comment:
I wonder if we shouldn't show the diff *before* sending the update? It would make things interactive but seems like that's an ok change to make?

Reply:
oh yeah I'm good with doing that, esp so users don't accidentally change their config without double checking

Reply:
Wonder if that can be addressed in another pr since it'll be adding an interactive component to updates

Reply:
Yea, a followup PR is good. Maybe we show them what's changing still before sending the update so the followup change is adding a prompt and not also change the order of output?